mmdet3d/core/visualizer/image_vis.py

```diff
@@ -120,6 +120,7 @@ def draw_lidar_bbox3d_on_img(bboxes3d,
     return plot_rect3d_on_img(img, num_bbox, imgfov_pts_2d, color, thickness)
 
 
+# TODO: remove third parameter in all functions here in favour of img_metas
```

Comment:
We keep this param because we want to keep the interface of all three projection functions the same, so that we can simply call it [here](https://github.com/open-mmlab/mmdetection3d/blob/c1f6bbab5582957d8ddfb487f9df02e81560e93e/mmdet3d/core/visualizer/show_result.py#L232).

Reply:
Yes I saw this reason, but other functions in this file use either `img_metas` or this third parameter. So we can probably drop one of them. I prefer to keep `img_metas`, with `lidar2img`, `depth2img` or `cam2img`. Am I missing something?

Reply:
Ah, I got it. I agree with you. We can do this in this PR?

Reply:
This PR affects only SUN RGB-D. If you do not mind I will refactor KITTI matrices and this function in the next PR.

Reply:
Sure. Thank you for your effort!